.circleci/config.yml

```
@@ -177,6 +177,7 @@ jobs:
     working_directory: ~/repo
     steps:
       - checkout
+      - VERSION=`cat VERSION`
```

Seems to cause build error. Maybe something like this would work?
```suggestion
      - run: VERSION=$(cat VERSION)
```

---

will do